src/domain/usecases/CopyProgramStageDataValuesUseCase.ts

```diff
@@ -167,9 +179,9 @@ function checkTargetDataValuesAreEmpty(events: ProgramEvent[], targetIds: Id[])
 
 export type CopyProgramStageDataValuesOptions = {
     programStageId: string;
-    dataElementIdPairs: [Id, Id][]; // [sourceDataElementId, targetDataElementId]
+    dataElementIdMappings: { source: Id; target: Id }[];
```

Comment:
don't change, but in that case where we have an object we typically use Array<...>